[2025-05-19T14:06:52.872+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_bank_reviews.run_clean_reviews manual__2025-05-19T11:20:46.404123+00:00 [queued]>
[2025-05-19T14:06:52.890+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_bank_reviews.run_clean_reviews manual__2025-05-19T11:20:46.404123+00:00 [queued]>
[2025-05-19T14:06:52.893+0100] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-05-19T14:06:52.926+0100] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): run_clean_reviews> on 2025-05-19 11:20:46.404123+00:00
[2025-05-19T14:06:52.943+0100] {standard_task_runner.py:60} INFO - Started process 81280 to run task
[2025-05-19T14:06:52.952+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'process_bank_reviews', 'run_clean_reviews', 'manual__2025-05-19T11:20:46.404123+00:00', '--job-id', '1925', '--raw', '--subdir', 'DAGS_FOLDER/post_processing.py', '--cfg-path', '/tmp/tmppq13vlm0']
[2025-05-19T14:06:52.958+0100] {standard_task_runner.py:88} INFO - Job 1925: Subtask run_clean_reviews
[2025-05-19T14:06:53.089+0100] {task_command.py:423} INFO - Running <TaskInstance: process_bank_reviews.run_clean_reviews manual__2025-05-19T11:20:46.404123+00:00 [running]> on host DESKTOP-RRUTCI2.
[2025-05-19T14:06:53.302+0100] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='process_bank_reviews' AIRFLOW_CTX_TASK_ID='run_clean_reviews' AIRFLOW_CTX_EXECUTION_DATE='2025-05-19T11:20:46.404123+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-19T11:20:46.404123+00:00'
[2025-05-19T14:06:53.310+0100] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-19T14:06:53.313+0100] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd ~/airflow/dbt_project && dbt run --select clean_bank_reviews']
[2025-05-19T14:06:53.341+0100] {subprocess.py:86} INFO - Output:
[2025-05-19T14:07:21.295+0100] {subprocess.py:93} INFO - [0m13:07:21  Running with dbt=1.8.7
[2025-05-19T14:07:24.787+0100] {subprocess.py:93} INFO - [0m13:07:24  Registered adapter: postgres=1.8.2
[2025-05-19T14:07:31.186+0100] {subprocess.py:93} INFO - [0m13:07:31  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-05-19T14:07:31.188+0100] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-05-19T14:07:31.190+0100] {subprocess.py:93} INFO - - models.dbt_project.forPy
[2025-05-19T14:07:32.066+0100] {subprocess.py:93} INFO - [0m13:07:32  Found 8 models, 1 seed, 4 data tests, 1 source, 540 macros
[2025-05-19T14:07:32.071+0100] {subprocess.py:93} INFO - [0m13:07:32
[2025-05-19T14:07:38.288+0100] {subprocess.py:93} INFO - [0m13:07:38  Concurrency: 4 threads (target='dev')
[2025-05-19T14:07:38.292+0100] {subprocess.py:93} INFO - [0m13:07:38
[2025-05-19T14:07:38.297+0100] {subprocess.py:93} INFO - [0m13:07:38  1 of 1 START sql view model public.clean_bank_reviews .......................... [RUN]
[2025-05-19T14:07:38.555+0100] {subprocess.py:93} INFO - [0m13:07:38  1 of 1 OK created sql view model public.clean_bank_reviews ..................... [[32mCREATE VIEW[0m in 0.25s]
[2025-05-19T14:07:38.607+0100] {subprocess.py:93} INFO - [0m13:07:38
[2025-05-19T14:07:38.609+0100] {subprocess.py:93} INFO - [0m13:07:38  Finished running 1 view model in 0 hours 0 minutes and 6.53 seconds (6.53s).
[2025-05-19T14:07:38.867+0100] {subprocess.py:93} INFO - [0m13:07:38
[2025-05-19T14:07:38.869+0100] {subprocess.py:93} INFO - [0m13:07:38  [32mCompleted successfully[0m
[2025-05-19T14:07:38.870+0100] {subprocess.py:93} INFO - [0m13:07:38
[2025-05-19T14:07:38.872+0100] {subprocess.py:93} INFO - [0m13:07:38  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-05-19T14:07:40.008+0100] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-19T14:07:40.130+0100] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=process_bank_reviews, task_id=run_clean_reviews, execution_date=20250519T112046, start_date=20250519T130652, end_date=20250519T130740
[2025-05-19T14:07:40.213+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-19T14:07:40.290+0100] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
